### PR TITLE
fix: Update text font color and weight 

### PR DESCRIPTION
**Pull request by DhariniJeeva**
This pr fixes the font color and weight to be in sync with the design  

Before:
Screen Shot 2021-07-09 at 10 56 59 AM

After: 
Screen Shot 2021-07-09 at 10 55 48 AM

### CODE REVIEW

File: ui/admin/app/styles/app.scss [hunk 1]
```diff
@@ -94,6 +94,8 @@
 .rose-table {
   p {
```
Comment (gsusmi) [inline ui/admin/app/styles/app.scss:95]: To reduce accidentally applying this style to all table columns, could this be restricted to just the header cells in a table?

**Review by gsusmi — COMMENTED** [review]
Minor note about nesting style to a header cell. Looks great otherwise! Ty!

File: ui/admin/app/styles/app.scss (hunk 1 above)
Comment (DhariniJeeva) [reply ui/admin/app/styles/app.scss:95]: good catch. updated!

**Review by calcaide — APPROVED** [review]
LGTM!

**Review by gsusmi — APPROVED** [review]

**Pull request merged**